packages/@aws-cdk/custom-resource-handlers/lib/aws-eks/cluster-resource-handler/cluster.ts

```diff
@@ -129,6 +129,52 @@ export class ClusterResourceHandler extends ResourceHandler {
       return this.onCreate();
     }
 
+    // Update tags
+    if (updates.updateTags) {
+      try {
+        //Describe cluster to get arn and to make sure Cluster exists
+        const cluster = (await this.eks.describeCluster({ name: this.clusterName })).cluster;
+        if (this.oldProps.tags) {
+          if (this.newProps.tags) {
+            //Update tags
+            const tagsToAdd = getTagsToUpdate(this.oldProps.tags, this.newProps.tags);
+            if (tagsToAdd) {
+              const tagConfig: EKS.TagResourceCommandInput = {
+                resourceArn: cluster?.arn,
```

Comment:
can there be any case where this value be empty or does not exist in which case we should add additional check to make sure we got the cluster arn value ?

Reply:
The describe is already in the try block, hence if there is any exception or cluster does not exist, it will throw `ResourceNotFoundException`. Apart from that we should get the cluster ARN at all times.